themes/default/content/templates/serverless-application/azure/index.md

```diff
@@ -0,0 +1,139 @@
+---
+title: "Azure Serverless Application"
+layout: template
+
+# Make sure this is description accurate for this template.
+meta_desc: The Azure Serverless Template makes it easy to deploy a serverless application on Azure with Pulumi, Azure Functions, and Azure Blob Storage.
+
+# Appears on the cards on template-overview pages.
+card_desc: Deploy a serverless application on Azure with Pulumi, Azure Functions, and Azure Blob Storage.
+
+# Used for generating language-specific links to templates on GitHub. (Example: `static-website-aws`)
+template:
+    prefix: serverless-azure
+
+# Used for generating links to sibling templates in the right-hand nav. Slug is this template's parent directory.
+cloud:
+  name: Azure
+  slug: azure
+
+# Be sure to replace this image. Figma source file:
+# https://www.figma.com/file/lGrSpwbGGmbixEuewMbtkh/Template-Architecture-Diagrams?node-id=15%3A196
+meta_image: meta.png
+
+# The content below is meant help you get started and to serve as a guide to work by. Feel free to adjust it needed for your template.
+---
+
+The Serverless Application template creates an infrastructure as code project in your favorite language that deploys a serverless application to Azure. It deploys an [Azure Storage Blob account]({{< relref "/registry/packages/azure-native/api-docs/storage/storageaccount" >}}) that's configured with a [container for static website hosting]({{< relref "/registry/packages/azure-native/api-docs/storage/storageaccountstaticwebsite" >}}) and an [Azure function]({{< relref "/registry/packages/azure-native/api-docs/web/webappfunction" >}}) that runs the business logic, which is written in the same language as the template. The application uses an [App Service plan]({{< relref "/registry/packages/azure-native/api-docs/web/appserviceplan" >}}) by default. The template ships with a placeholder website that displays the current time to give you a working Pulumi project out of the box that you can customize easily and extend to suit your needs.
```

Comment:
This first paragraph's starting to feel a bit wall-of-texty to me. Could we remove the "in your favorite language" bit? I'm not sure it's really adding all that much value here (you can see the actual list of supported languages a few lines down), and in general I'd like for this first paragraph to be lean and scannable:

```suggestion
The Serverless Application template creates an infrastructure as code project that deploys a serverless application to Azure with Pulumi. It deploys an [Azure Blob Storage account]({{< relref "/registry/packages/azure-native/api-docs/storage/storageaccount" >}}) configured for [static website hosting]({{< relref "/registry/packages/azure-native/api-docs/storage/storageaccountstaticwebsite" >}}) and an [Azure Function]({{< relref "/registry/packages/azure-native/api-docs/web/webappfunction" >}}) written in the same language as the Pulumi program. The template ships with placeholder content to give you a working project out of the box that you can customize easily and extend to suit your needs.
```

Also suggest removing the line about app-service plans, as I don't believe you _can_ deploy an Azure Function App without one (so probably not worth calling out -- at least not in this first paragraph).